document/CoreFormat.mdk

```diff
@@ -11,8 +11,10 @@ that is, it shall consist of a comma-separated sequence of name/value pairs, enc
 as described in the JSON specification.
 We refer to the object represented by this top-level value as the `sarifLog` object (&sect;[#sarifLog]).
 
-Every JSON property name in the SARIF format shall be a camelCase name.
-This includes the names of properties defined in property bags such as `result.properties` (&sect;[#result-properties]).
+Every JSON property name defined by the SARIF format shall be a camelCase name.
+Because the names of properties defined in property bags (&sect;[#property-bag-properties]) such as `result.properties` (&sect;[#result-properties]),
+are not defined by the SARIF format, they are not subject to this requirement.
+These property names should also be cameCase, but see [Annex C](#Annex-C) for exceptions.
```

Comment:
typo, should be camel-case. where is the guidance that converters in particular should be sure not to mess with casing from original native tool formats? we have two scenarios for property bag population:

1) direct producer adds a property. we recommend using a camel-cased name here?
2) converter populates with a property. we require that converters preserve original name/casing